internal/parser/testdata/complex.toml

```
@@ -53,12 +54,18 @@ NoNodes = 7
 NodesType = "PC"
 Networks = ["Wifi 2.4GHZ", "Wifi 5GHz", "Hubbed", "Switched"]
 MovementModel = "static"
+PredefinedPosition = true
+
+[NodeGroup.Position]
+X = 40
+Y = 50
```

Please try to stick to the style used in the other examples:
```suggestion
Position.X = 40
Position.Y = 50
```

---

This style you refer to does not appear in any other .toml file

---

It does, for example [here](https://github.com/stg-tud/bp2022_netlab/blob/main/examples/complex.toml#L83), [here](https://github.com/stg-tud/bp2022_netlab/blob/main/examples/multiple_networks_and_groups.toml#L15) and [here](https://github.com/stg-tud/bp2022_netlab/blob/main/internal/parser/testdata/complex.toml#L8)